pkg/validation/internal/operatorhub_test.go

```diff
@@ -255,6 +255,12 @@ func TestCheckBundleName(t *testing.T) {
 			warnStrings: []string{"csv.metadata.Name memcached-operator.v1.3 is not following the " +
 				"versioning convention (MAJOR.MINOR.PATCH e.g 0.0.1): https://semver.org/"},
 		},
+		{
+			name:        "should return a error when the bundle name is > 63",
```

Comment:
The operatorhub io validator is an optional validator if I'm not mistaken. Do we want to enforce this name length rule with the CSV validator instead? If any user will end up with a broken on-cluster experience if the name of their CSV is too long maybe this should go under the default CSV or bundle validator.